Jenkinsfile

```diff
@@ -165,12 +165,20 @@ def generateStep(version, tav = ''){
       } catch(e){
         error(e.toString())
       } finally {
-        docker.image('node:11').inside("-v ${WORKSPACE}/${BASE_DIR}:/app"){
-          sh(label: "Convert Test results to JUnit format", script: 'cd /app && .ci/scripts/convert_tap_to_junit.sh')
+        runNodejs(){
+          dir("${BASE_DIR}"){
+            sh(label: "Convert Test results to JUnit format", script: '.ci/scripts/convert_tap_to_junit.sh')
+          }
         }
         junit(allowEmptyResults: true, keepLongStdio: true, testResults: "${BASE_DIR}/**/junit-*.xml")
         codecov(repo: 'apm-agent-nodejs', basedir: "${BASE_DIR}", secret: "${CODECOV_SECRET}")
       }
     }
   }
 }
+
+def runNodejs(Closure body){
+  docker.image('node:11').inside(){
```

Comment:
Does the `11` part of `node:11` mean Node.js version 11?

If so we might want to use either v10 or v12 as those are or will be LTS and is considered more stable (I know this might not be part of this PR, but just wanted to mention it as I saw it here 😄)

Reply:
ok, 12 in that case